presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -379,14 +379,13 @@ public PrestoSparkQueryRunner(
         }
         pluginManager.installPlugin(new HivePlugin("hive", Optional.of(metastore)));
 
-        Map<String, String> properties = ImmutableMap.<String, String>builder()
-                .put("hive.allow-drop-table", "true")
-                .put("hive.allow-rename-table", "true")
-                .put("hive.allow-rename-column", "true")
-                .put("hive.allow-add-column", "true")
-                .put("hive.allow-drop-column", "true")
-                .putAll(hiveProperties).build();
-        connectorManager.createConnection("hive", "hive", properties);
+        Map<String, String> properties = new HashMap<>(hiveProperties);
+        properties.putIfAbsent("hive.allow-drop-table", "true");
```

Comment:
@pdabre12 : Any reason we have not added the rest of the properties viz. hive.allow-rename-(table)column, hive.allow-add-column, hive.allow-drop-column to NativeQueryRunnerUtils.getNativeWorkerHiveProperties() function above ?

Reply:
I guess the need was only for `hive.allow-drop-table`. But good point. We could add the remaining and remove them from here.

Reply:
Yup , the need was only for `hive.allow-drop-table`. 
My plan was to add the other properties once this refactor was accepted.

Reply:
Then likely `getNativeWorkerHiveProperties` is not the right place to add this.
Can we add inside at `presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java:225`?